src/essreflectometry/amor/normalize.py

```diff
@@ -2,12 +2,14 @@
 # Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
 import scipp as sc
 
-from ..reflectometry import orso
+# from ..reflectometry import orso
+from ..types import Normalized, NormalizedIofQ, Reference, Sample
 
 
 def normalize_by_supermirror(
```

Comment:
Remind me why this is AMOR-specific?

Reply:
It probably isn't. Ill move it to the generic reflectometry module.